scripting/get5/teamlogic.sp

```diff
@@ -279,33 +283,31 @@ Get5Team CSTeamToGet5Team(int csTeam) {
   }
 }
 
-Get5Team GetAuthMatchTeam(const char[] steam64) {
+Get5Team GetAuthMatchTeam(const char[] steam64, bool includeCoaches = true, bool includePlayers = true) {
```

Comment:
`GetMatchTeamFromAuth` maybe? This current call makes it sound like you're retrieving the steam ID from the MatchTeam.